src/dist/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 add_subdirectory(failure_detector)
 add_subdirectory(replication)
 add_subdirectory(cluster_scheduler)
-add_subdirectory(deployment_service)
+#add_subdirectory(deployment_service)
```

Comment:
gpid or get_gpid should be in the base class as service_app. Derived classes do not need to implement that.